starboard/BUILD.gn

```diff
@@ -57,9 +57,13 @@ group("gn_all") {
     deps += [ "//starboard/benchmark" ]
   }
 
-  if (build_with_separate_cobalt_toolchain && is_cobalt_toolchain) {
-    assert(sb_is_modular,
-           "sb_is_modular should be set when building with cobalt_toolchain")
+  if (build_with_separate_cobalt_toolchain) {
+    if (is_cobalt_toolchain) {
+      assert(sb_is_modular,
+             "sb_is_modular should be set when building with cobalt_toolchain")
+    }
+
+    deps += [ "//starboard/contrib/cast/cast_starboard_api/samples:cast" ]
```

Comment:
You would need a new variable. We don't want all modular builds to have cast in them.
Something like `cast_sample`?

Reply:
GN variables should follow https://gn.googlesource.com/gn/+/main/docs/style_guide.md#Naming-conventions. If we introduce a variable for this, it should likely be prefixed with `enable_` or `use_`

Reply:
Added `use_contrib_cast`